src/main.rs

```diff
@@ -318,7 +318,7 @@ fn run_command(vm: &VirtualMachine, source: String) -> PyResult<()> {
 
 fn run_module(vm: &VirtualMachine, module: &str) -> PyResult<()> {
     debug!("Running module {}", module);
-    vm.import(module, &vm.ctx.new_tuple(vec![]), 0)?;
+    vm.import(module, &vm.ctx.empty_tuple(), 0)?;
```

Comment:
Please do not use this empty tuple directly, and make it a private method on `PyContext`. I think it is better to call the new_tuple function with an empty vector, so we have one way to create a tuple.